_docs/collection-details.md

```diff
@@ -205,7 +205,10 @@ Once the funds have been successfully received from the sender, `payin_method.pa
 To initiate a XOF mobile collection, please use the following details (`phone_number` used below are examples):
 At the moment, we only support collections in:
 
-- Senegal(Orange) - this requires the sender to request an OTP auth code via USSD `#144#391*ORANGE_MONEY_PIN_CODE#` . The auth code retrieved should be included in the `otp` parameter in `in_details`.
+- Senegal(Orange) - this requires the sender to request an OTP auth code via USSD `#144#391*ORANGE_MONEY_PIN_CODE#` . The auth code retrieved should be included in the `otp` parameter in `in_details`. Uses `ussd_voucher` in `ux_flow` parameter.
+- Ivory Coast(Orange) - this requires the sender to request an OTP auth code via USSD `#144*82#` or `#144#391*ORANGE_MONEY_PIN_CODE#`. The auth code retrieved should be included in the `otp` parameter in `in_details`. Uses `ussd_voucher` in `ux_flow` parameter.
```

Comment:
`- Ivory Coast(Orange) - this requires the sender to request an OTP auth code via USSD `#144*82#`. The auth code retrieved should be included in the `otp` parameter in `in_details`. Uses `ussd_voucher` in `ux_flow` parameter.`
```suggestion
- Ivory Coast(Orange) - this requires the sender to request an OTP auth code via USSD `#144*82#`. The auth code retrieved should be included in the `otp` parameter in `in_details`. Uses `ussd_voucher` in `ux_flow` parameter.
```